type_templates/reverb.js

```diff
@@ -0,0 +1,37 @@
+import * as THREE from 'three';
+import metaversefile from 'metaversefile';
+const {useApp, useCleanup, getReverbZone} = metaversefile;
+
+
+export default e => {
+  const app = useApp();
+  let worldReverbZone = getReverbZone();
+  const srcUrl = ${this.srcUrl};
+  const mode = app.getComponent('mode') ?? 'attached';
+  let j = null;
+  let reverbZoneInfo = null;
+  if (mode === 'attached') {
+    (async () => {
+      const res = await fetch(srcUrl);
+      j = await res.json();
+      reverbZoneInfo = j.args;
+      if (j) {
+        worldReverbZone.push(reverbZoneInfo);
```

Comment:
I would use an add/remove api, not hack the array directly. It will make debugging and future additions much easier.